app/src/main/java/com/teamagam/gimelgimel/app/message/AlertMessageTextFormatterData.java

```diff
@@ -3,20 +3,20 @@
 import android.content.Context;
 import com.teamagam.gimelgimel.R;
 import com.teamagam.gimelgimel.domain.alerts.entity.Alert;
-import com.teamagam.gimelgimel.domain.messages.AlertMessageTextCreator;
+import com.teamagam.gimelgimel.domain.messages.AlertMessageTextFormatter;
 import javax.inject.Inject;
 
-public class AlertMessageTextCreatorImpl implements AlertMessageTextCreator {
+public class AlertMessageTextFormatterData implements AlertMessageTextFormatter {
```

Comment:
Names should make sense.
Use one of the following:
`DataAlertMessageTextFormatter` , `AlertMessageTextDataFormatter`, etc...
I'd go with the first....

Reply:
done